tox.ini

```
@@ -41,6 +41,7 @@ commands =
     format: black -l 80 --include \.pct\.py$ --check notebooks
     # tests (run with "-- --runslow yes" to run all tests)
     tests: pip install . -r tests/requirements.txt -c tests/constraints.txt
```

You can put it all on this line:

```suggestion
    tests: pip install .[qhsri] -r tests/requirements.txt -c tests/constraints.txt
```

Though if you follow the suggestion for updating `generate_constraints.sh`, changing this file might not be necessary at all (as the qhsri requirements ill be in constraints.txt).